README.md

```diff
@@ -4,28 +4,28 @@ GitHub Action to find and output the number of last updated issue that has given
 
 ### Action inputs
 
-| Name            | Description                                                                                                                                                                                                                                                                                                     | Default  |
-| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
-| **\*** `labels` | Comma or newline-separated list of labels that the issue must have                                                                                                                                                                                                                                              |
-| `state`         | Issue state to filter by. Can be one of the following strings: <ul><li> <code>"open"</code>: if you want to look up for open issues only </li><li> <code>"closed"</code>: if you want to look up for closed issues only </li><li> <code>"all"</code>: if you want to look up for open or closed ones </li></ul> | `"open"` |
+| Name            | Description                                                                                                                                                                                                                                                                                                     | Default                    |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| **\*** `repository`      | The target GitHub owner and name separated by slash. For example: `micalevisk/last-issue-action`.                                                                                                                                                                                                               | `${{ github.repository }}` |
+| **\*** `token`           | A [`repo`][settings-create-scoped-token] scoped [Personal Access Token][docs-token-pat] with at least [`issues: read` permissions][docs-token-permissions].                                                                                                                                                     | `${{ github.token }}`      |
```

Comment:
I think I made a mistake here too. `token` and `repository` are meant to be `gh-token` and `gh-repository`?

I added the `gh-` prefix in `action.yml` because you didn't seem to like the more generic/ambiguous name `token` I think? Do you want to stick without the prefix as the README shows here, or keep using the `gh-` prefix?

Reply:
following the common names on official actions, keep `token` and `repository`